dids/module.md

```diff
@@ -73,7 +73,7 @@ fun main() {
   val keyManager = InMemoryKeyManager()
   keyManager.import(jsonKeySet)
 
-  val did = DidKey(uri = didUri, keyManager = keyManager)
+  val did = DidKey.load(did = didUri, keyManager = keyManager)
```

Comment:
nit: reads slightly confusingly because the object returned is called `did`, but the argument you pass to load is also called `did`

Reply:
It's because it _is_ confusing :D 

I've rename to `uri`.